src/payment/strategies/braintree/braintree.ts

```diff
@@ -5,6 +5,8 @@ import { PaypalAuthorizeData, PaypalButtonOptions, PaypalButtonRender, PaypalSDK
 
 import { VisaCheckoutInitOptions, VisaCheckoutPaymentSuccessPayload, VisaCheckoutTokenizedPayload } from './visacheckout';
 
+export const BRAINTREE_COMPONENTS = ['buttons', 'messages'];
```

Comment:
Defined components in fact belongs to PayPal:
https://github.com/paypal/paypal-messaging-components
so i think we should name them like `PAYPAL_COMPONENTS`

Reply:
Renamed